packages/appeals-service-api/package.json

```diff
@@ -21,7 +21,7 @@
 		"test": "jest",
 		"test:cov": "jest --coverage --verbose",
 		"test:watch": "jest --watchAll --runInBand",
-		"integration-test": "npm run db:generate && cross-env NODE_OPTIONS=--experimental-vm-modules jest --config ./jest.developer-test.config.js",
+		"integration-test": "npm run db:generate && cross-env NODE_OPTIONS=--experimental-vm-modules jest --silent --config ./jest.developer-test.config.js",
```

Comment:
does this leave the test pass/fail statuses?

Reply:
yeah, and the pino logging is still there, this just disables console